EventFilter/EcalRawToDigi/plugins/UnpackGPU.cu

```diff
@@ -1,4 +1,5 @@
 #include "EventFilter/EcalRawToDigi/interface/ElectronicsIdGPU.h"
+#include "EventFilter/EcalRawToDigi/interface/DCCRawDataDefinitions.h"
```

Comment:
nice :+1:

Reply:
I did not replace all masks with the ones defined in the header since I thing this is out of the scope of this patch.